src/commands/nogps/DriveToGate.cpp

```diff
@@ -15,7 +16,13 @@ command_t DriveToGate::getOutput()
 	{
 		double angle = atan2(post.y(), post.x());
 		double vel = (post.topRows<2>().norm() <= 2 * radius) ? slowVel : fastVel;
-		return {angle * thetaKP, vel};
+		double targthvel = angle * thetaKP;
+		double maxthvel = 0.5;
+        std::cout << "Requesting vel " << vel << std::endl;
```

Comment:
Indentation is a little funky here
```suggestion
		std::cout << "Requesting vel " << vel << std::endl;
```